vllm/utils/hashing.py

```diff
@@ -61,3 +62,20 @@ def get_hash_fn_by_name(hash_fn_name: str) -> Callable[[Any], bytes]:
         return sha256_cbor
 
     raise ValueError(f"Unsupported hash function: {hash_fn_name}")
+
+
+def safe_hash(data: bytes, usedforsecurity: bool = False) -> HASH:
```

Comment:
```suggestion
def safe_hash(data: bytes, usedforsecurity: bool = True) -> HASH:
```

I think `True` is a safer default, and it matches Python's default.